modules/@apostrophecms/translation/index.js

```diff
@@ -27,10 +29,12 @@ module.exports = {
 
           // We don't support multiple providers yet, so just use the first one
           // when no provider is specified.
-          const providerName = self.getProvider(translationProvider)?.name;
-          const module = self.getProviderModule(name);
+          const { name: providerName, module } = self.getProvider(translationProvider);
+          const manager = self.apos.modules[module];
```

Comment:
This part was unclear, because we tried to get the proxy provider with actual existing provider (like deepl) which make no sense if we want the provider.
We could just use the `automatic-translation` module like I did below.